src/low-level/smtp/mailsmtp_types.h

```diff
@@ -70,6 +70,7 @@ enum {
   MAILSMTP_ERROR_AUTH_ENCRYPTION_REQUIRED,
   MAILSMTP_ERROR_STARTTLS_TEMPORARY_FAILURE,
   MAILSMTP_ERROR_STARTTLS_NOT_SUPPORTED,
+  MAILSMTP_ERROR_CLIENTID_NOT_SUPPORTED,
```

Comment:
Could you move it to the end of the emum?

Reply:
Would you also like the IMAP error in mailimap_types.h moved?

> MAILIMAP_ERROR_CLIENTID